cmd/yggctl/main.go

```diff
@@ -250,6 +250,33 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:        "message-journal",
+			Usage:       "Display a list of all worker messages and emitted events",
+			UsageText:   "yggctl message-journal",
+			Description: "The message-journal command sends a dbus call to yggdrasil to retrieve a message journal containing all worker messages and emitted events.",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "truncate-message",
+					Aliases:  []string{"t"},
+					Usage:    "Truncates worker event messages if they exceed the specified character count (10 by default).",
```

Comment:
A default value can automatically be included in the help output by using the `Value` field of a Flag. That sets the default value returned by `c.String("truncate-message")` as well, so you don't need to check the flag value later.